services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy

```diff
@@ -0,0 +1,261 @@
+package org.openkilda.functionaltests.spec.northbound.switches
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.messaging.info.meter.SwitchMeterEntries
+import org.openkilda.testing.Constants
+import org.openkilda.testing.model.topology.TopologyDefinition
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch
+import org.openkilda.testing.service.northbound.NorthboundService
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.client.HttpClientErrorException
+import spock.lang.Ignore
+import spock.lang.Issue
+import spock.lang.Narrative
+import spock.lang.Shared
+import spock.lang.Unroll
+
+@Narrative("The test suite checks if traffic meters, including default, are set and deleted in a correct way.")
+class MetersSpec extends BaseSpecification {
+
+    static DISCO_PKT_RATE = 200 // Number of packets per second for the default flows
+    static DISCO_PKT_SIZE = 250 // Default size of the discovery packet
+    static NOVI_BURST_MIN = 1.004 // Guaranteed minimum flow burst rate on a Noviflow switch, with rounding errors.
+    static NOVI_BURST_MAX = 1.051 // Possible maximum flow burst rate on a Noviflow switch, with rounding errors.
+    static DISCO_PKT_BURST = 4096 // Default desired packet burst rate for the default flows (ignored by Noviflow)
+
+    @Autowired
+    TopologyDefinition topology
+
+    @Autowired
+    NorthboundService northboundService
+
+    @Shared
+    def centecs
+
+    @Shared
+    def nonCentecs
```

Comment:
I don't understand the point of these two shared variables. 1 - you never actually use them. 2 - they get initialized in a very strange way when calling `get..()` method. This is very unnatural.